backend/src/v5/middleware/dataConverter/inputs/teamspaces/projects/models/metadata.js

```diff
@@ -56,8 +65,11 @@ const generateSchema = (existingMetadata) => {
 Metadata.validateUpdateCustomMetadata = async (req, res, next) => {
 	try {
 		const { teamspace, container, metadata } = req.params;
-		const containerMetadata = await getMetadataById(teamspace, container, metadata, { metadata: 1 });
-		const schema = generateSchema(containerMetadata.metadata);
+		const existingMetadata = await getMetadataByQuery(teamspace, container, { _id: metadata },
+			{ _id: 0, metadata: 1 });
+		const nonCustomMetadataLookup = existingMetadata.metadata.filter((m) => m.custom !== true)
```

Comment:
I don't think you need a filter if you're using reduce? just return `a`  if it's a custom data.